src/services/MeasureService.ts

```diff
@@ -88,4 +89,34 @@ export class MeasureService implements Service<fhir4.Measure> {
 
     return createMeasurePackageBundle(measure[0]);
   }
+
+  /**
+   * result of sending a POST or GET request to:
+   * {BASE_URL}/4_0_1/Measure/$data-requirements or {BASE_URL}/4_0_1/Measure/:id/$data-requirements
+   * creates a Library with all data requirements for the specified measure
+   * requires parameters id and/or url and/or identifier, but also supports version as supplemental (optional)
+   */
+  async dataRequirements(args: RequestArgs, { req }: RequestCtx) {
+    logger.info(`${req.method} ${req.path}`);
+    logger.info('Using package to create measure bundle');
+    const measureBundle = await this.package(args, { req: req });
```

Comment:
Adding a logger statement about using the package operation is a good idea 👍 I did notice that when passing `req` into `this.package`, the `package` logger statement says the `req.path` is data requirements instead of package:

```
{"level":"info","message":"GET /4_0_1/Measure/ColorectalCancerScreeningsFHIR/$data-requirements"}
{"level":"info","message":"Using package to create measure bundle"}
{"level":"info","message":"GET /4_0_1/Measure/ColorectalCancerScreeningsFHIR/$data-requirements"}
{"level":"info","message":"Assembling collection bundle from Measure ColorectalCancerScreeningsFHIR"}
```
This makes it look like data requirements is being called twice. Maybe we should pass in an updated path to `this.package` so that its logger is accurate

Reply:
Hmm- things seem to go a bit sideways when I start futzing with the path... I can experiment with it more, but did you have any more thoughts on implementation you imagine?

Reply:
If it's too complicated to fix then I'd say we can leave this since it's such a small thing. I tried this out and it seemed to work alright:

``` javascript
const packagePath = req.path.replace(/data-requirements/i, 'package');
const measureBundle = await this.package(args, { req: { ...req, path: packagePath } });
```
which returned `"GET /4_0_1/Measure/ColorectalCancerScreeningsFHIR/$package"`

Reply:
oh interesting, I tried to directly futz with req.path and got that it was readonly, but yeah, this seems to work. Thank you for digging further!

Reply:
Oh- also I like your replacement approach better than mine which was just `req.path.replace('data-requirements', 'package');` :-)